### ISSUE TITLE
Question - Set Sizing Performance effect

### ISSUE THREAD

**Issue by mohsenomidi** [oktal/pistache#595]
@dennisjenkins75 

there are some constant settings in Pistache for handling the Request/Response and buffers, etc

If I change the default values and set it to the bigger numbers dose this changes make the performance issues or other side effects? 

```
    static constexpr size_t MaxBacklog = 128; ???
    static constexpr size_t MaxEvents  = 1024; ???
    static constexpr size_t MaxBuffer  = 4096; ????
    static constexpr size_t DefaultWorkers = 1;
    static constexpr size_t DefaultTimerPoolSize = 128;
    static constexpr size_t DefaultMaxRequestSize = 4096; ???
    static constexpr size_t DefaultMaxResponseSize = std::numeric_limits<uint32_t>::max();
    static constexpr size_t ChunkSize  = 1024;

```

**Comment by dennisjenkins75** [comment]
Hello Mohsen,

   I do not know.  IIUC, Pistache was created to run on a small system, handle lots of small requests very quickly.  However, Pistache should scale well to larger systems and larger payloads.  I've not tinkered with those tunables, and my own use-case is does not demand high-performance, so I've not paid any attention to them.  The only thing that I changed was the server-side max upload size.  I needed to push a few 100KiB, the default was 4 KiB.
